src/celengine/dsodb.cpp

```diff
@@ -71,14 +71,14 @@ DSODatabase::~DSODatabase()
 DeepSkyObject* DSODatabase::find(const uint32_t catalogNumber) const
```

Comment:
could you use `AstroCatalog::IndexNumber` instead of `uint32_t` here and in other cases?